csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -161,6 +161,7 @@ public struct OrtApi
         public IntPtr GetStringTensorElement;
         public IntPtr FillStringTensorElement;
         public IntPtr AddSessionConfigEntry;
+        public IntPtr SetSessionProgrammingProjection;
```

Comment:
you can't add this here.   it changs the vtable and breaks versioning.   also ; shouldn't we bump the version for this change?  ORT_API_VERSION